code/modules/research/designs/medical.dm

```diff
@@ -140,6 +150,16 @@
 	category = "Medical"
 	build_path = /obj/item/weapon/circular_saw/plasmasaw
 
+/datum/design/diamond_surgicaldrill
+	name = "Diamond Surgical Drill"
+	desc = "A true extension of the surgeon's body, this marvel instantly cuts the organ, clamp any bleeding, and retract the skin, allowing for the immediate commencement of therapeutic steps."
```

Comment:
Still has the incision manager's `desc`.

Reply:
I knew I forgot something in the redo of this. I will fix it.